StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/ReadabilityResources.resx

```diff
@@ -522,6 +522,21 @@
   <data name="SA1139CodeFix" xml:space="preserve">
     <value>Use literal suffix notation instead of casting</value>
   </data>
+  <data name="SA1135CodeFix" xml:space="preserve">
+    <value>Qualify using directive</value>
+  </data>
+  <data name="SA1135Description" xml:space="preserve">
+    <value>All using directives must be qualified.</value>
```

Comment:
(Feel free to resolve this by filing a follow-up issue.)

❕ This feature needs to be updated to account for the change in wording from *must* to *should*.

Reply:
✔️